gamemode/modules/inventory/sv_inventory_commands.lua

```diff
@@ -1,4 +1,4 @@
-local function giveItem( caller, target, item, count )
+function RVR.Commands.giveItem( caller, target, item, count )
     if count < 1 then
```

Comment:
Theres a point to be made about making these commands global, however they are already global within the Commands module, and this PR is not the place to change this due to the inconsistency it'd cause with other modules.

Reply:
nerd